graph/api/src/main/java/org/jboss/windup/graph/service/GraphService.java

```diff
@@ -280,6 +280,34 @@ protected T getUnique(Traversal<?, ?> query)
         return frame((Vertex)resultObj);
     }
 
+    public T getOrCreate(String key, String value, String... kvs)
```

Comment:
Please add a test for this method.
`GraphServiceTest` seems a good place to add tests for this method.
For sure a test to check there's no duplication of model and just one is created.
Please also add a test when an odd number of parameters is provided in input because it looks like this code will break: it would be better to check `kvs` has even size as first thing and in case manage properly if it's odd.